docs/changelog.rst

```diff
@@ -19,6 +19,7 @@ Changelog
 * :feature:`-` Swaps made via uniswap v3 auto routers (both v1 and v2) will now be decoded correctly.
 * :feature:`3231` Optimism is now supported. Optimism balances will be shown and optimism transactions will be decoded.
 * :feature:`1756` Uniswap and sushiswap pool join/exit events are now properly decoded and taken into account during PnL report.
+* :bug:`-` Now customized ownership proportions of validators owned by eth1 addresses will be respected.
```

Comment:
I have corrected this multiple times. Do not start a sentence with `Now`.

```suggestion
* :bug:`-` Customized ownership proportions of validators owned by eth1 addresses will now be properly respected.
```

Reply:
Ah(((